enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/BackButton/BackButton.tsx

```diff
@@ -25,13 +25,18 @@ export type InteractiveQuestionBackButtonProps = Omit<
 export const BackButton = ({
   ...actionIconProps
 }: InteractiveQuestionBackButtonProps) => {
-  const { onNavigateBack } = useInteractiveQuestionContext();
+  const { onNavigateBack, backToDashboard } = useInteractiveQuestionContext();
```

Comment:
I don't feel like I want to use the name `dashboardOverride` on the question context as it's not describing what it does really well.